qemu/tests/virtio_fs_nfs_migration.py

```diff
@@ -0,0 +1,279 @@
+import re
+
+from virttest import env_process, error_context, nfs, utils_disk, utils_misc, utils_test
+
+from provider import virtio_fs_utils
+from provider.storage_benchmark import generate_instance
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Basic migration test with diffirent cache modes and i/o over nfs
+    Steps:
+        1. Setup a local nfs server and mount it to dir fs and targetfs
+        2. Run the virtiofsd daemon to share fs with different cache modes
+        3. Boot the source guest with the virtiofs device in step2
+        4. Mount the virtiofs targets inside the guest
+        5. Create a file and get its md5, then run fio
+        6. Run the virtiofsd daemon to share targetfs
+        7. Boot the target guest with the virtiofs device in step6
+        8. Do migration from the source guest to the target guest
+        9. No error occurs, the virtiofs is mounted automatically and
+           the file md5 should keep the same, fio is still running
+
+    :param test: QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+
+    def create_service(session):
+        if os_type == "windows":
+            error_context.context("Create virtiofs service in guest.", test.log.info)
+
+            driver_name = params["driver_name"]
+            session = utils_test.qemu.windrv_check_running_verifier(
+                session, vm, test, driver_name
+            )
+            viofs_svc_name = params["viofs_svc_name"]
+            virtio_fs_utils.create_viofs_service(
+                test, params, session, service=viofs_svc_name
+            )
+        return session
+
+    def delete_service():
+        if os_type == "windows":
+            error_context.context("Delete virtiofs service in guest.", test.log.info)
+            session = vm.wait_for_login()
+            virtio_fs_utils.delete_viofs_serivce(test, params, session)
+            session.close()
+
+    def start_service(session):
+        def start_multifs_instance(fs_tag, fs_target, fs_volume_label):
```

Comment:
The same, no multifs.

Reply:
Talked with Tingting, I will add multifs later